.github/workflows/jira-creation.yml

```diff
@@ -21,8 +21,8 @@ jobs:
   call-creation-action:
     uses: dbt-labs/actions/.github/workflows/jira-creation.yml@main
     with:
-      project_key: "CT"
+      project_key: "SEMANTIC"
     secrets:
       JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
```

Comment:
@callum-mcdata also I don't have permissions to see this, but are these 3 repo secrets present in Settings? Maybe that is the auth error you are seeing